string FileName
string DataPath
int32 Lap
RoiList[] Rois
bool TestVideo

